frontend/src/interfaces/context.ts

```diff
@@ -4,6 +4,8 @@ export interface IUnleashContextDefinition {
     createdAt: string;
     sortOrder: number;
     stickiness: boolean;
+    usedInProjects?: number;
```

Comment:
can we remove the optionals in the long term? I understand that in a transitional period we may do expand/contract so optionality may be needed.